### PR TITLE
chore(tests): add openshift checker extension installation test using…

### PR DESCRIPTION
**Pull request by odockal**
… OCI image URL

### What does this PR do?
Add e2e test that verifies the openshift checker installation via OCI image url.

### What issues does this PR fix or reference?
#8029 

### How to test this PR?
`yarn install;yarn test:e2e:extension`

It will run more tests at the end. expecially using openshift checker.

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

**Comment by odockal** [comment]
E2E tests workflow run with this branch: https://github.com/containers/podman-desktop/actions/runs/9886497857

File: tests/playwright/src/specs/extension-installation.spec.ts [hunk 1]
```diff
@@ -83,19 +91,40 @@ describe.each([
...
-    const extensionCatalog = new ExtensionCatalogCardPage(page, extensionType);
-    await playExpect(extensionCatalog.parent).toBeVisible();
-
-    await playExpect.poll(async () => await extensionCatalog.isInstalled()).toBeFalsy();
-    await extensionCatalog.install(180000);
-
-    await extensionsPage.openInstalledTab();
-    await playExpect.poll(async () => await extensionsPage.extensionIsInstalled(extensionLabel)).toBeTruthy();
-  }, 200000);
+  test.runIf(extensionType.indexOf(OPENSHIFT_LOCAL) >= 0 || extensionType.indexOf(OPENSHIFT_SANDBOX) >= 0)(
```
Comment (cbr7) [inline tests/playwright/src/specs/extension-installation.spec.ts]: I'm not sure I understand what the point of this is

File: tests/playwright/src/specs/extension-installation.spec.ts [hunk 2]
```diff
@@ -83,19 +91,40 @@ describe.each([
...
+      await playExpect.poll(async () => await extensionCatalog.isInstalled()).toBeFalsy();
+      await extensionCatalog.install(180000);
+
+      await extensionsPage.openInstalledTab();
+      await playExpect.poll(async () => await extensionsPage.extensionIsInstalled(extensionLabel)).toBeTruthy();
+    },
+    200000,
+  );
+
+  test.runIf(extensionType.indexOf(OPENSHIFT_CHECKER) >= 0)(
```
Comment (cbr7) [inline tests/playwright/src/specs/extension-installation.spec.ts]: Same as above.

File: tests/playwright/src/specs/extension-installation.spec.ts (hunk 1 above)
Comment (odockal) [reply tests/playwright/src/specs/extension-installation.spec.ts]: Different extension, different check. Given we have one test file, not all tests can be done for all extensions. Ie. openshift local and sandbox have provider cards on dashboard page, while openshift checker does not.

Comment (cbr7) [reply tests/playwright/src/specs/extension-installation.spec.ts]: No, that I understand, what I dont get is why didn't you just check `extensionType === OPENSHIFT_LOCAL`  or whatever, I dont understand this use of indexOf >= 0

Comment (odockal) [reply tests/playwright/src/specs/extension-installation.spec.ts]: this is the whatever. Seems like development of the condition, first it was just searching for substring, later it became constant. so I just kept it there.

Comment (odockal) [reply tests/playwright/src/specs/extension-installation.spec.ts]: triple equals sign placed.

**Review by cbr7 — APPROVED** [review]

**Pull request merged**